hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/components/SPUIComponentProvider.java

```diff
@@ -206,6 +208,45 @@ public static Label createNameValueLabel(final String label, final String... val
         return nameValueLabel;
     }
 
+    /**
+     * Method to CreateName value labels.
+     *
+     * @param label
+     *            as string
+     * @param values
+     *            as string
+     * @return HorizontalLayout
+     */
+    public static HorizontalLayout createNameValueLayout(final String label, final String... values) {
+        final String valueStr = StringUtils.arrayToDelimitedString(values, " ");
+
+        final Label nameValueLabel = new Label("<b>" + label + "</b>");
```

Comment:
I was able to do this, but I reverted it again. The problem is, that two font-weight properties would be in competition (font-weight: bold vs font-weight: 300). If i use "!important" with the "bold"-one, then text appears bigger that the not bold text. Because of that I reverted it and would prefer to use it how it is, to keep the design as it was before.

Reply:
Removed the bold tags as discussed.